tests/playwright/src/bootc-extension.spec.ts

```diff
@@ -43,6 +43,7 @@ const imageTag = 'stream9';
 const extensionName = 'bootc';
 const extensionLabel = 'redhat.bootc';
 const isLinux = os.platform() === 'linux';
+const isWindows = os.platform() === 'win32';
```

Comment:
this seems like nice code to be shared across all the e2e tests. Not a scope of this PR, just something to think of for podman-desktop/tests-playwright contributions.